File: WebAPI/WebAPI.Models/Basic/WorkSpace.cs
using System;

namespace WebAPI.Models.Basic
{
    public class WorkSpace
    {
        public Guid WorkSpaceId { get; set; }
        
        public string WorkSpaceName { get; set; }
        
        public string WorkSpaceDescription { get; set; }
        
        public DateTime CreationDate { get; set; }
    }
}

File: web/src/mappers/workspace.ts
import { IWorkSpace } from '../types/workspace';

export const mapToWorkSpaceModel = (data): IWorkSpace => ({
    workSpaceId: data.workSpaceId,
    workSpaceName: data.workSpaceName,
    workSpaceDescription: data.workSpaceDescription,
    creationDate: new Date(data.creationDate),
});
